### PR TITLE
Use scripts to generate repos

### PR DESCRIPTION
**Pull request by lars-petter-hauge**
Idea is to have scripts create the repos. The templates should be flawless, and the scripts will add any bugs that later commits are to fix.

Still need some work to fix all paths (moved things around a bit)

### CODE REVIEW

**Comment by lars-petter-hauge** [comment]
Scripts should be working as expected now, need to verify the status of each example

Also missing one of the examples (focused on the ones with the same story)

**Comment by lars-petter-hauge** [comment]
There's a diff on example 3, are the included grammatical errors intended? 

```
diff --color examples/example_3/Commas_are_important examples_orig/example_3/Commas_are_important
1c1
< Once upon a time, there was a family of wolves who lived deep in
---
> Once upon a time, there was a family of birds who lived deep in
9c9
< matters into their own hands and hunt for their meal. They
---
> maters into their own hands and hunt for their meal. They
```

So pretty much the example is about merge conflicts, but the `Commas_are_important` includes the same grammatical errors as in the previous examples..

I've left them out of the example script for now.

**Comment by lars-petter-hauge** [comment]
Also example 4:

```
diff --color examples/example_4/Commas_are_important examples_orig/example_4/Commas_are_important
3c3
< reunion where grandma wolf would be joining them. As they
---
> reunion where grandma bird would be joining them. As they
```

The example has 3 fixup commits, one of them are `change birds -> wolves`. The original structure had another error which wasn't changed. Is it supposed to be that way, or should I have both errors in there and change both?

**Comment by lars-petter-hauge** [comment]
I've gone through the examples now, they should all be identical (except for the issues listed above).

**Comment by lars-petter-hauge** [comment]
All scripts fixed, verified in codespaces. Added the missing example as well.

File: scripts/create_example_4.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+commit_msg="Add example
+
+In this example we will be doing an interactive rebase, this is
+typically done when we want to rewrite history. For example when
+we have a draft PR and we want to make it ready for review. We want
+to create a narrative which makes it easier for the reviewer, and
+also creates a good history in the repository.
+
+In this example there are two branches, main and fix_bugs, your
+task is to rebase fix_bugs onto main, but change the order of the
```
Comment (oyvindeide) [inline scripts/create_example_4.sh:24]: We talked about doing this in two steps, first rebasing on self, then rebasing on main. Know this is just a port of existing behavior, so can change in another PR if you prefer.

**Review by oyvindeide — COMMENTED** [review]
Really nice work! Makes it much more readable, as shown by the fact that I could make a comment to the workflow in one of the examples :)

**Review by oyvindeide — APPROVED** [review]

**Pull request merged**